src/test/java/seedu/sellsavvy/logic/commands/personcommands/EditPersonCommandTest.java

```diff
@@ -73,6 +79,30 @@ public void execute_someFieldsSpecifiedUnfilteredList_success() {
         assertCommandSuccess(editPersonCommand, model, expectedMessage, expectedModel);
     }
 
+    @Test
+    public void execute_editedPersonInDisplay_success() {
```

Comment:
Maybe this can be renamed to `execute_PersonEditedWithOrderDisplayed_success()`?
I'm not very sure what this is supposed to test so I'm assuming it's just testing whether editing a person whose order is displayed does not change the OrderList being displayed/the selectedPerson.

Reply:
this one makes sure that when u update the person already displayed it change the one in the wrapper ObjectProperty so next time someone edit, it will edit correct one and also retoggle gui.

Reply:
fixed the naming